client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -69,8 +70,10 @@ export default React.createClass( {
 							beforeNavigate={ this.updateScrollPosition } />;
 		}
 
+		// TODO: should be refactored into separate components
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
I don't particularly like that we've disabled the warning here. The issue remains, and should be shown as a warning, not disabled.